charts/cp-schema-registry/values.yaml

```diff
@@ -80,9 +80,39 @@ resources: {}
   #  cpu: 100m
   #  memory: 128Mi
 
+datadog:
+  enabled: true
+  annotations:
+    ad.datadoghq.com/cp-schema-registry-server.check_names: '["openmetrics"]'
+    ad.datadoghq.com/cp-schema-registry-server.init_configs: '[{}]'
+    ad.datadoghq.com/cp-schema-registry-server.logs: '[{"source":"schema-registry","service":"{{ .Release.Name }}"}]'
+    ad.datadoghq.com/cp-schema-registry-server.instances: |
+      [{
+      "prometheus_url": "http://%%host%%:5556/metrics",
+      "namespace": "schema_registry",
+      "metrics": [
+        "cp_kafka_schema_registry_jetty_metrics_connections_active",
+        "cp_kafka_schema_registry_jetty_metrics_connections_opened_rate",
+        "cp_kafka_schema_registry_jetty_metrics_connections_closed_rate",
+        "cp_kafka_schema_registry_jersey_metrics_request_error_rate",
+        "cp_kafka_schema_registry_master_slave_role",
+        "cp_kafka_schema_registry_jersey_metrics_request_byte_rate",
+        "cp_kafka_schema_registry_jersey_metrics_request_latency_avg",
+        "cp_kafka_schema_registry_jersey_metrics_request_size_avg",
+        "jvm*"
+      ],
+      "type_overrides": {
+        "*": "gauge",
+        ".+_total": "counter"
+      }
+      }]
+  labels:
+    team: "team_dbre"
+
 ## Custom pod annotations
 podAnnotations:
   cluster-autoscaler.kubernetes.io/safe-to-evict: "true"  # Allow the autoscaler to downscale the cluster.
+  sidecar.istio.io/inject: "false"
```

Comment:
Is it really wanted to disable istio in this PR about datadog ?

Also, why should we disable Istio on this app ? (please write a few words of explainations in a comment in the yaml, for future readers to understand)

Reply:
> much better globally (because same than what we decided before and already applied elsewhere).

@frivoire I humbly disagree with the quoted sentence. 
I don't think that things are "much better" just because they respect a convention, even if this convention was decided in a collegial way, it can be discussed and if possible improved or even removed, above I said what I thought of the formatting with the helpers file; I find that it doesn't bring much for the reasons I cited, **that can be discussed of course**.
Aymeric answered me that he found it more readable, fair enough, even if I don't agree it remains a subjective opinion.
 
But just saying that it's much better because it respects the rule, that I don't agree. We need to respect the rule of course, but we can discuss the rule and remove it or evolve it if necessary/possible.  
 
By the way, this rule is not applicable everywhere, even for dbre products, for example, I think we cannot apply it for products managed by an operator, since we can't control in our chart the kube resources manifests (deployments / pods /etc.) 
Here is the Kafka connect chart, since we cannot change the pods manifests ourselves we put directly the annotations in the appropriate section of the CRD 
https://github.com/blablacar/helm-charts/blob/master/dbre/strimzi-kafka-connect/values.yaml#L64

Reply:
> Is it really wanted to disable istio in this PR about datadog ?

Yes actually I noticed errors on the agent when I tried to integrate datadog, I tried to disable the istio proxy and it worked like a charm, I should've put the comment before you're right 👍 

I will add it. 

Here I just reproduced the issue, some logs for the record 
      Metric Samples: Last Run: 0, Total: 0
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 1, Total: 1
      Average Execution Time : 78ms
      Last Execution Date : 2022-03-02 12:39:03 UTC (1646224743000)
      Last Successful Execution Date : Never
      Error: 502 Server Error: Bad Gateway for url: http://10.96.6.102:5556/metrics
      Traceback (most recent call last):
        File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/base.py", line 1017, in run
          self.check(instance)
        File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/base_check.py", line 137, in check
          self.process(scraper_config)
        File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/mixins.py", line 556, in process
          for metric in self.scrape_metrics(scraper_config):
        File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/mixins.py", line 486, in scrape_metrics
          response = self.poll(scraper_config)
        File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/base/checks/openmetrics/mixins.py", line 829, in poll
          response.raise_for_status()
        File "/opt/datadog-agent/embedded/lib/python3.8/site-packages/requests/models.py", line 943, in raise_for_status
          raise HTTPError(http_error_msg, response=self)
      requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: http://10.96.6.102:5556/metrics

Reply:
@OuesFa : sorry if my message was taken as too rude.
Of course, I totally agree that every convention can be discussed if needed, and they also have limits and can't be applied everywhere.

Reply:
No no it was really to answer "sur le fond".  
I know your message was not meant to be rude.